src/backend/cdb/cdbutil.c

```diff
@@ -4030,6 +4038,9 @@ getgpsegmentCount(void)
 	/* 1 represents a singleton postgresql in utility mode */
 	int32 numsegments = 1;
 
+	if (getgpsegmentCount_hook)
```

Comment:
It seems duplicated due to error git merge.
How about remove this duplicated function?